$NetBSD: patch-configure.ac,v 1.6 2014/10/13 21:52:50 ryoon Exp $

* Accept non-bash statement, and fix Makefile generation
* Use gstreamer under OpenBSD
* Add FreeBSD support (GNU patch is not needed)
* Detect pkgsrc apache-ant library
* Fix installation directory

--- configure.ac.orig	2014-09-16 20:10:41.000000000 +0000
+++ configure.ac
@@ -72,7 +72,7 @@ AbsolutePath()
     if test -d "$rel" ; then
         cd "$rel" || AC_MSG_ERROR([absolute path resolution failed for "$rel".])
         absolute_path="$(pwd)"
-        cd - &> /dev/null
+        cd - > /dev/null
     else
         AC_MSG_ERROR([Failed to resolve absolute path.  "$rel" does not exist or is not a directory.])
     fi
@@ -651,6 +651,8 @@ aix*)
 
 openbsd*)
     test_gtk=yes
+    build_gstreamer=yes
+    build_gstreamer_0_10=yes
     test_tde=yes
     test_kde=yes
     test_freetype=yes
@@ -6567,7 +6569,7 @@ if test \( "$_os" != "WINNT" -o "$WITH_M
 
     # Assume visibility is not broken with libc++. The below test is very much designed for libstdc++
     # only.
-    if test "$HAVE_GCC_VISIBILITY_FEATURE" = "TRUE" -a "$CPP_LIBRARY" = GLIBCXX; then
+    if test "$HAVE_GCC_VISIBILITY_FEATURE" = "TRUE" -a "$CPP_LIBRARY" = GLIBCXX -a "$COM_GCC_IS_CLANG" != "TRUE"; then
         dnl gcc#19664, gcc#22482, rhbz#162935
         AC_MSG_CHECKING([if STL headers are visibility safe (GCC bug 22482)])
         AC_EGREP_HEADER(visibility push, string, stlvisok=yes, stlvisok=no)
@@ -8066,7 +8068,7 @@ no|disable)
     else
         # Unset variables set by the above AM_PATH_PYTHON so that
         # we actually do check anew.
-        unset PYTHON am_cv_pathless_PYTHON ac_cv_path_PYTHON am_cv_python_version am_cv_python_platform am_cv_python_pythondir am_cv_python_pyexecdir
+        #unset PYTHON am_cv_pathless_PYTHON ac_cv_path_PYTHON am_cv_python_version am_cv_python_platform am_cv_python_pythondir am_cv_python_pyexecdir
         AM_PATH_PYTHON([3.3],, [:])
         if test "$PYTHON" = :; then
             AC_MSG_RESULT([internal])
@@ -8123,9 +8125,9 @@ if test $enable_python = system; then
     elif test "$cross_compiling" != yes; then
         # Unset variables set by the above AM_PATH_PYTHON so that
         # we actually do check anew.
-        unset PYTHON am_cv_pathless_PYTHON ac_cv_path_PYTHON am_cv_python_version am_cv_python_platform am_cv_python_pythondir am_cv_python_pyexecdir
+        #unset PYTHON am_cv_pathless_PYTHON ac_cv_path_PYTHON am_cv_python_version am_cv_python_platform am_cv_python_pythondir am_cv_python_pyexecdir
         # This causes an error if no python command is found
-        AM_PATH_PYTHON([3.3])
+        AM_PATH_PYTHON([2.6])
         python_include=`$PYTHON -c "import distutils.sysconfig; print(distutils.sysconfig.get_config_var('INCLUDEPY'));"`
         python_version=`$PYTHON -c "import distutils.sysconfig; print(distutils.sysconfig.get_config_var('VERSION'));"`
         python_libs=`$PYTHON -c "import distutils.sysconfig; print(distutils.sysconfig.get_config_var('LIBS'));"`
@@ -9928,8 +9930,8 @@ if test -z "$PATCH"; then
     AC_MSG_ERROR([\"patch\" not found in \$PATH, install it])
 fi
 
-dnl On Solaris, FreeBSD or MacOS X, check if --with-gnu-patch was used
-if test "$_os" = "SunOS" -o "$_os" = "FreeBSD" -o "$_os" = "Darwin"; then
+dnl On Solaris, or MacOS X, check if --with-gnu-patch was used
+if test "$_os" = "SunOS"  -o "$_os" = "Darwin"; then
     if test -z "$with_gnu_patch"; then
         GNUPATCH=$PATCH
     else
@@ -12194,7 +12196,7 @@ EOF
     dnl Checking for ant.jar
     if test "$ANT_HOME" != "NO_ANT_HOME"; then
         AC_MSG_CHECKING([Ant lib directory])
-        if test -f $ANT_HOME/lib/ant.jar; then
+        if test -f $ANT_HOME/lib/java/ant/ant.jar; then
             ANT_LIB="$ANT_HOME/lib"
         else
             if test -f $ANT_HOME/ant.jar; then
@@ -12625,7 +12627,7 @@ AC_SUBST(MANDIR)
 DOCDIR=[$(eval echo $(eval echo $docdir))]
 AC_SUBST(DOCDIR)
 
-INSTALLDIR="$LIBDIR/$INSTALLDIRNAME"
+INSTALLDIR="$PREFIXDIR"
 AC_SUBST(INSTALLDIR)
 
 TESTINSTALLDIR="${BUILDDIR}/test-install"
